obspy/io/stationxml/tests/test_stationxml.py

```diff
@@ -114,6 +115,41 @@ def test_read_and_write_minimal_file(self):
         self._assert_station_xml_equality(file_buffer,
                                           expected_xml_file_buffer)
 
+    def test_subsecond_read_and_write_minimal_file(self):
+        """
+        Test reading and writing of sub-second time in datetime field,
+        using creation time
+
+        """
+        filename = os.path.join(self.data_dir,
+                                "minimal_station_with_microseconds.xml")
+        inv = obspy.read_inventory(filename)
+
+        # Write it again. Also validate it to get more confidence. Suppress the
+        # writing of the ObsPy related tags to ease testing.
+        file_buffer = io.BytesIO()
+
+        # XXX helper variable to debug writing the full random file, set True
+        # XXX for debug output
+        write_debug_output = False
```

Comment:
This is IMHO not needed. It is quite unlikely that this test will fail in the future and as such this adds a bit of noise to the test case. Then you can also always validate and remove the debug saving which will also increase test coverage ;-) Its easy enough to add debug statements whenever one has to debug something.